Makefile

```diff
@@ -62,3 +62,11 @@ generate-hive-templates: generate-oauth-templates
 		${GEN_POLICY_CONFIG};\
 		${GEN_TEMPLATE}; \
 	fi
+
+.PHONY: enforce-backplane-rules
+enforce-backplane-rules:
```

Comment:
Does this makerule need to get added to the default set on L40?

Reply:
It was requested in slack to leave the checks disabled in this PR. I would recommend that we add this make target to `default` with #1590 to align enforcement and conformance in the same commit.

Reply:
@cblecker it could become a default rule after https://github.com/openshift/managed-cluster-config/pull/1590 merges or as a part of that PR (it isn't right now).  But not in this PR as there are a ton of things that will not pass.